docs/eventing/getting-started.md

```diff
@@ -0,0 +1,503 @@
+    ```sh
+    < HTTP/1.1 202 Accepted
+    < Content-Length: 0
+    < Date: Mon, 12 Aug 2019 19:48:18 GMT
+    ```
+
+
+3. To make the third request, which creates an event that has the `type` `greeting` and the`source` `sendoff`, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-hello-goodbye" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: greeting" \
+      -H "Ce-Source: sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello Knative! Goodbye Knative!"}'
+   ```
+
+   Result: When the event is sent to the `Broker`, the `Triggers` `hello-display` and `goodbye-display` will activate and send it to the event consumer of the same name.
+
+   If the event has been received, you will receive a `202 Accepted` response similar to the one below:
+    ```sh
+    < HTTP/1.1 202 Accepted
+    < Content-Length: 0
+    < Date: Mon, 12 Aug 2019 19:48:18 GMT
+    ```
+
+4. Exit SSH by typing `exit` into the command prompt.
+    
+If everything has been done correctly, you should have sent 2 events to the `hello-display` event consumer and 2 events to the `goodbye-display` event consumer (note that `say-hello-goodbye` is sent to *both* `hello-display` and `goodbye-display`). You will verify that these events were received correctly in the next section.
+
+## Verifying events were received 
+
+After sending events, verify that your events were received by the appropriate `Subscribers`.
+
+1. Look at the logs for the `hello-display` event consumer by running the following command:
+
+    ```sh
+    kubectl --namespace event-example logs -l app=hello-display --tail=-1
+    ```
+
+
+    Result: The command shows the `Attributes` and `Data` of the events you sent to `hello-display`:
+
+    ```sh
+    ☁️  cloudevents.Event
+    Validation: valid
+    Context Attributes,
+      specversion: 0.2
+      type: greeting
+      source: not-sendoff
+      id: say-hello
+      time: 2019-05-20T17:59:43.81718488Z
+      contenttype: application/json
+    Extensions,
+      knativehistory: default-broker-srk54-channel-24gls.event-example.svc.cluster.local
+    Data,
+      {
+        "msg": "Hello Knative!"
+      }
+    ☁️  cloudevents.Event
+    Validation: valid
+    Context Attributes,
+      specversion: 0.2
+      type: greeting
+      source: sendoff
+      id: say-hello-goodbye
+      time: 2019-05-20T17:59:54.211866425Z
+      contenttype: application/json
+    Extensions,
+      knativehistory: default-broker-srk54-channel-24gls.event-example.svc.cluster.local
+    Data,
+     {
+       "msg": "Hello Knative! Goodbye Knative!"
+     }
+    ```
+
+2. Look at the logs for the `goodbye-display` event consumer by running the following command:
+
+    ```sh
+    kubectl --namespace event-example logs -l app=goodbye-display --tail=-1
```

Comment:
These commands still only show me that "10 line" shortened response. Whats missing so that I see the same verbose examples that are shown here?

Reply:
I changed the --tail flag to show 100 lines, and it was more verbose. According to kubectl documentation, tail=-1 is supposed to show all, and it's not working for me again. But tail=100 works, so I'll keep that for now.